gradle.properties

```diff
@@ -1,2 +1,2 @@
 #Wed, 23 Jan 2019 12:03:29 +0100
-version=0.1-SNAPSHOT
+version=10.0.0-SNAPSHOT
```

Comment:
you will have to change the version on both branches
master -> 10.1.0-SNAPHOST
10.0.X -> 10.0.0-rc1